px-dropdown-trigger.html

```diff
@@ -124,6 +124,30 @@
       icon: {
         type: String,
         value: ''
+      },
+      /**
+       * If set, the items list will be in read only mode, the list is disabled
+       * and the user can not click the items. No events are fired.
+       */
+      disabledViewable: {
+        type: Boolean,
+        observer: '_setDisabledViewableClass'
+      },
+      /**
+       * If set, the items list will be in read only mode, the list is disabled
+       * and the user can not click the items. No events are fired.
+       */
+      readOnly: {
+        type: Boolean,
+        observer: '_setReadOnlyClass'
+      },
+      /**
+       * If set, the items list will be in read only mode, the list is disabled
```

Comment:
Is the dropdown considered to be in 'read only mode' in this case? The user will not even be able to see the list of items if its disabled right?

Reply:
there are three separate cases - `read-only`, `disabled-viewable`, and `disabled`. If multiple cases are set, then the component should prioritize which is more important in the following order:

- disabled
- disabledViewable
- read-only

Reply:
Right, makes sense, I'm just curious about the language being concise since these options have at least confused me.

Also, @arongraham-ge I guess the test cases should also reflect this priority logic.